sdk/spring/azure-security-jca/src/main/java/com/azure/azure/security/jca/implementation/http/model/SecretBundle.java

```diff
@@ -1,62 +1,28 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.security.keyvault.jca.model;
+package com.azure.azure.security.jca.implementation.http.model;
 
 import java.io.Serializable;
 
-/**
- * The SecretBundle REST model.
- */
 public class SecretBundle implements Serializable {
```

Comment:
But java doc like this is pointless:
```
/**
 * Get Id.
 */
public getId() {
    ...
}
```
Since the pipeline is passed, I think it's OK to delete such pointless java doc.